src/Microsoft.AspNetCore.Mvc.Razor/Compilation/CompiledViewManifest.cs

```diff
@@ -12,13 +11,7 @@ public static class CompiledViewManfiest
     {
```

Comment:
I'm going to get this in and clean up precompilation and follow up on this today. 😄